src/woo_publications/publications/api/serializers.py

```diff
@@ -122,6 +146,14 @@ def validate_publicatiestatus(
 
         return value
 
+    def validate_documenthandelingen(self, value: document_actions):
+        if len(value) != 1:
+            raise serializers.ValidationError(
+                _("You can only have one documenthandeling at a time.")
+            )
+
```

Comment:
I think you can pass `max_length` and `min_length` kwargs to enforce the count.

However, we need to check this with ICATT - I thought [we had agreed](https://github.com/GeneriekPublicatiePlatformWoo/registratie-component/issues/76#issuecomment-2419576422) they don't have to send anything and we will make sure there is a default. Something for standup.

If the client sends nothing, we set up the default (which the model already takes care of with the model field defaults).